[fix] バージョン修正

sEmoto0808 opened the pull request:
Swift3対応

Commit:
[fix] バージョン修正

stv-ekushida commented on README.md:
```diff
@@ -1,7 +1,7 @@
 
 # バージョン
-v1.0.0
+v1.0.1
```
メジャーバージョンアップなので、v2.0.0で良いのでは？

Commit:
[modify] Swift3に対応するために以下を修正
・コーディング規約のバージョン
・アクセス制御にfileprivateを追加
・AnyObject → Any に修正
・ファイルフォーマットのファイル内順序を修正
・NSURL→URL、NSDate→Date、CGColor→cgColor
